ch04.md

```diff
@@ -148,7 +148,7 @@ Refactor to remove all arguments by partially applying the functions.
   
 {% initial src="./exercises/ch04/exercise_b.js#L3;" %}  
 ```js  
-const filterQs = xs => filter(x => match(/q/i, x), xs);
+const filterQs = xs => filter(x => test(/q/i, x), xs);
```

Comment:
There is no `test` function in _support/index.js_, so the above will fail without a definition.

Reply:
my bad. I hadn't seen the _support/index.js_ file and assumed the match definition was either the following as used little above the exercises section:

`var match = curry(function(what, str) {
  return str.match(what);
});`

or `_.match` from rambda that was missing an import. as implicated by: 

> We'll use a library called Ramda which curries every function by default.

I can close the PR or rename `match` to `test` as you suggested. I skimmed through Part 1 of the book and did not see any references to _support/index.js_ , I'd be willing to add those if they are indeed missing and not a mistake on my part. cheers!

Reply:
Actually, when you run the solutions, you will use _exercises/support.js_ via `npm run ch04` etc, which has better console output.

I like the idea of renaming `match` to `test` and change the examples but that is just my opinion and I have little to say. I'm reading the book, just like you.

About the Ramda remark, it is a little curious since Ramda is never used in any of the examples in the book. Perhaps, it was a thought to include Ramda at some point but I really like implementations here because they are coded to show how the theory is applied and not performance, which seem to be a higher goal for Ramda (with good reason).